MSAL/src/public/MSALPublicClientApplication.h

```diff
@@ -420,7 +420,7 @@
                                 representing the TenantID property of the directory)
  @param  claimsRequest          The claims parameter that needs to be sent to token endpoint. When claims
                                 is passed, access token will be skipped and refresh token will be tried.
- @param  forceRefresh           Ignore any existing access token in the cache and force MSAL to
+ @param  forceRefresh           Ignore any existing access token in the cache and force MSAL/Broker to
```

Comment:
Do we need to expose brokers in public comments or to remove the `MSAL` directly?

Reply:
clarified offline